tests/monitoring/component_monitoring.go

```diff
@@ -224,7 +225,7 @@ var _ = Describe("[Serial][sig-monitoring]Component Monitoring", Serial, decorat
 			err = virtClient.RbacV1().ClusterRoleBindings().Delete(context.Background(), "kubevirt-controller", metav1.DeleteOptions{})
 			Expect(err).ToNot(HaveOccurred())
 
-			vmi := tests.NewRandomVMI()
+			vmi := libvmi.New()
```

Comment:
There is another usage at line 243.
Also, `libvmi.New()` will produce a different vmi respect to `tests.NewRandomVMI()`.
The diff will be in:
- libvmi.WithInterface(libvmi.InterfaceDeviceWithMasqueradeBinding())
- libvmi.WithNetwork(v1.DefaultPodNetwork())
- libvmi.WithResourceMemory(...)

Are those fields not required for this test?

Reply:
`libvmi.WithResourceMemory("1Mi")` is required for the VMI to start (without any guest). I don't think the network is necessary here.

(I got a reminder of the minimum memory on https://github.com/kubevirt/kubevirt/pull/11302 )